### DISCUSSION TITLE
Speedtest Server Finder

### DISCUSSION THREAD

**Discussion by jiriteach** [alexjustesen/speedtest-tracker#1602] [Ideas]
@alexjustesen - More an FYI. With the removal of the UI to find Speedtest servers, I ended up building a quick Speedtest server finder - https://github.com/jiriteach/speedtest-server-finder.  Useful utility to share with others. I do have it hosted on Cloudflare Pages but it not public. Allows you to quickly grab the recommended server or search for others without needing to do this via cli.

I could make the hosted site public if there is interest else it could also be added into the image - just havent got around to do a PR.

**Comment by alexjustesen** [comment]
This is awesome! Let's connect because I am also building a tool to list and validate servers now that I have the base site setup.

**Comment by mattish91** [comment]
I found my servers from here: https://williamyaps.github.io/wlmjavascript/servercli.html

It's pretty good, i just had to search for my country and got a list, it seams to be working fine.

**Reply by bozzio83** [reply]
How do you specify which server to use?

**Reply by alexjustesen** [reply]
You can specify the server using env variables, docs ref https://docs.speedtest-tracker.dev/getting-started/environment-variables#speedtest
